db/comdb2.h

```diff
@@ -1252,18 +1253,14 @@ struct ireq {
     /* REGION 1 */
     /************/
     char *frommach; /* machine that the buffer came from */
```

Comment:
Thanks. Moved a couple of things around, but alas did not result in any savings.